frontend/src/components/suggestions/SuggestionsList.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC, useState } from 'react';
+import { Suggestion } from './types';
+import { SuggestionItem } from './SuggestionItem';
+import { Typography, Box, Checkbox, Button, makeStyles, createStyles, Theme } from '@material-ui/core';
+import { Playlist } from '../playlists/types';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    actions: {
+      '& button': {
+        marginLeft: theme.spacing(2)
+      }
+    }
+  })
+);
+
+interface SuggestionsListProps {
+  suggestions: Suggestion[];
```

Comment:
Are we planning to use Redux to manage the state ?

Reply:
Yes, I'm planning to setup redux in the next PR, or maybe make the most of the apollo client, since we're gonna use that on the backend. But I don't have an extended experience on Apollo Client, it's the opportunity to learn I guess ?